apps/cli-gui/src/app/home/home.component.ts

```diff
@@ -1,18 +1,20 @@
+import { AsyncPipe } from '@angular/common';
 import { ChangeDetectionStrategy, Component } from '@angular/core';
-import { MatSidenavModule } from '@angular/material/sidenav';
 import { RouterOutlet } from '@angular/router';
-import { MenuComponent } from '@angular-cli-gui/ui';
+import { SidenavComponent, SidenavService } from '@angular-cli-gui/ui';
 
 import { HOME_MENU_ITEMS } from './home.consts';
 
 @Component({
   selector: 'cli-home',
   standalone: true,
-  imports: [MatSidenavModule, MenuComponent, RouterOutlet],
+  imports: [SidenavComponent, RouterOutlet, AsyncPipe],
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class HomeComponent {
   menuItems = HOME_MENU_ITEMS;
+
+  constructor(readonly sidenavService: SidenavService) {}
```

Comment:
You're not injecting it `private/public`, it's on purpose?

Reply:
it's public by default cuz I'm using it in the template